src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1860,6 +1904,21 @@ export function getCNAColorByAlteration(alteration: string): string {
     }
 }
 
+export function transformMutatedType(eventType: string): string {
+    if (eventType === undefined || eventType === 'NA') return eventType;
+
+    // Split the input string by underscores
+    var words = eventType.split('_');
+
+    // Capitalize the first letter of each word
+    var capitalizedWords = words.map(function(word) {
```

Comment:
i think lodash has _.capitalize that does this

Reply:
updated